tox.ini

```
@@ -31,10 +35,21 @@ usedevelop = True
 
 [testenv:metadata-validation]
 deps =
-    collective.checkdocs
-    twine
+    collective.checkdocs>=0.2
+    pep517 >= 0.5.0
+    pip>=19.1.1
+    setuptools>=39.0
+    twine>=1.13.0
+    wheel>=0.31.0
+
 usedevelop = False
 # Ref: https://twitter.com/di_codes/status/1044358639081975813
 commands =
-    python -m setup checkdocs check --metadata --restructuredtext --strict --verbose
-    twine check .tox/dist/*
+    rm -rf {toxinidir}/dist
```

This is not cross-platform. Better do
```suggestion
    python -c "import path; path.Path('{toxinidir}/dist').rmtree_p()"
```

---

> This is not cross-platform. Better do

While 3rd party module are you referring to because this is clearly not part of standard library. I am not sure if for the sake of portability we need to introduce a new dependency.

Can you please elaborate? What is the library? Is it available for py27...py37?

---

`path.py` and yes, it works on 2.7